pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -58,7 +58,14 @@
 , runCommandLocal
 , makeWrapper
 }:
-
+let
+  # run gsettings with desktop schemas for using in "kcm_accces" kcm
```

Comment:
Here

Reply:
thanks for pointing out 
forced pushed now